services/steam/steam-workshop.service.js

```diff
@@ -223,14 +223,6 @@ class SteamFileService extends BaseSteamAPI {
   async onRequest({ response }) {
     throw new Error(`onRequest() wasn't implemented for ${this.name}`)
   }
-
-  static get defaultBadgeData() {
-    return { label: 'steam' }
-  }
-
-  static get category() {
-    return 'other'
-  }
```

Comment:
This default label is not used for any of the steam badge. I've set `category` in the subclasses below.